controllers/managersController.js

```diff
@@ -20,4 +32,58 @@ const managersSignup = async (req, res) => {
     });
   }
 };
-export default managersSignup;
+export const managersGet = async (req, res) => {
+  try {
+    const manager = await Managers.find();
+    res.status(200).json({
+      message: 'success',
+      manager,
+    });
+  } catch (err) {
+    console.log(err);
+    res.status(500).json({
+      message: 'failed to get all managers',
+    });
+  }
+};
+
+export const login = async (req, res) => {
```

Comment:
not getting any response on login

Reply:
response now is provided, thanks anyways